packages/forc/VERSION

```diff
@@ -1 +1 @@
-0.49.3
+0.53.0
```

Comment:
> [!WARNING]
> This will likely need to be released in unison together with:
> - https://github.com/FuelLabs/fuels-ts/pull/2021
> - https://github.com/FuelLabs/fuels-ts/pull/2044
> 
> cc @Torres-ssf